package.json

```diff
@@ -10,7 +10,7 @@
 	"author": "",
 	"license": "MIT",
 	"contributors": [
-		{ "name": "D.C. Rockwell", "email": "dcautomates@gmail.com" }
+		{ "name": "D.C. Rockwell", "email": "dcautomates@gmail.com" "name": "Ethan Klein" "email": "ethansklein@yahoo.com" }
```

Comment:
You need to modify this line to:

```
{ "name": "D.C. Rockwell", "email": "dcautomates@gmail.com"},
{ "name": "Ethan Klein", "email": "ethansklein@yahoo.com" }
```

Then commit and push to your local branch. It will automatically update here on the pull request.